### ISSUE TITLE
[BUG] Application crashes while changing duration using seekto

### ISSUE THREAD

**Issue by do4Mother** [bsutton/sounds#43]
## Sounds Version : 

- **Prerelease** version

- 1.0.0-beta13

- **FULL**

----------------------------------------------------------

## Severity

- Crash

--------------------------------------------------------

## Platforms you faced the error 

- iOS

----------------------------------------------

## Logs

- can't capture logs

-----------------------------------------------

## flutter doctor :

```
[✓] Flutter (Channel stable, 1.20.2, on Mac OS X 10.15.5 19F101, locale en-ID)
    • Flutter version 1.20.2 at /Users/do4mother/Development/flutter
    • Framework revision bbfbf1770c (5 weeks ago), 2020-08-13 08:33:09 -0700
    • Engine revision 9d5b21729f
    • Dart version 2.9.1

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/do4mother/Development/Android
    • Platform android-30, build-tools 29.0.2
    • ANDROID_HOME = /Users/do4mother/Development/Android
    • Java binary at:
      /Library/Java/JavaVirtualMachines/jdk1.8.0_251.jdk/Contents/Home/bin/java
    • Java version Java(TM) SE Runtime Environment (build 1.8.0_251-b08)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4.1, Build version 11E503a
    • CocoaPods version 1.9.3

[!] Android Studio (not installed)
    • Android Studio not found; download from
      https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup
      for detailed instructions).

[✓] VS Code (version 1.49.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.14.1

[✓] Connected device (1 available)
    • iPhone 8 (mobile) • 9DBE5509-EE12-4DAB-A66E-8713CB80D798 • ios •
      com.apple.CoreSimulator.SimRuntime.iOS-13-4 (simulator)

! Doctor found issues in 1 category.
```
-----------------------------------------------

**Describe the bug**
I tried implementing code from the documentation where it refers to https://bsutton.gitbook.io/sounds/monitoring, but after I try to move the slider the app always crashes. it only happens on iOS.

**To Reproduce**
1. create new project
2. copy paste the code
3. run the project
4. move the slider
5. the application is crashed and no logs can be retrieved

-----------------------------------------------

**Comment by relf108** [comment]
We're currently in the midst of an overhaul of all the code providing functionality on iOS. We are getting closer by the day to being done with this but its taking some time. When we do release this code it is likely to fix this issue and if the issue persists we should have an easy time fixing it. Sorry for the delay!

**Comment by SudoSharma** [comment]
@relf108 Any update on this? Thanks for all the great work!

**Comment by relf108** [comment]
We are still working on the issue but we've hit some snags and have decided to use FFI to manage the iOS side of the project. This he caused some delays but we think it will be better for the project in the long run. Additionally, I've unfortunately been bogged down with other work but I'm hoping to get back to working in this full time in the next fortnight or so.
Thanks so much for your patience!

**Comment by bsutton** [comment]
Can you please test against the just released 1.0.0
